integreat_cms/cms/templates/pages/_page_machine_translation_overlay.html

```diff
@@ -0,0 +1,101 @@
+{% load i18n %}
+<div id="machine_translation_overlay"
+     data-hix-threshold="{{ hix_threshold }}"
+     class="hidden fixed items-center justify-center inset-0 bg-opacity-75 bg-gray-800 z-50 cursor-pointer">
+    <div class="cursor-auto justify-center w-[500px] max-h-fit px-[10px] z-50 m-auto">
+        <div class="bg-white opacity-100 content rounded shadow-md w-full">
+            <div class="flex items-center w-full rounded p-4 bg-water-500">
+                <h2 class="font-bold font-default">
+                    <i icon-name="bot" class="pb-1"></i>
+                    {% translate "Machine translation" %}
+                </h2>
+            </div>
+            <div class="w-full p-4 rounded shadow overflow-scroll max-h-[80vh]">
+                <span class="block">{% translate "This report shows a summary of your attempted machine translation. Please confirm if everything is correct" %}</span>
+                <div class="mt-4">
+                    <h3>
+                        <b>{% translate "Language selection" %}</b>
+                    </h3>
+                    <div class="flex items-center mt-2">
+                        <div class="flex">
+                            <span class="fp fp-rounded fp-{{ source_language.primary_country_code }} w-7 mr-2"></span>
+                            <p>{{ source_language }}</p>
+                        </div>
+                        <i class="mx-4" icon-name="arrow-big-right"></i>
+                        <div class="flex">
+                            <span class="fp fp-rounded fp-{{ language.primary_country_code }} w-7 mr-2"></span>
+                            <p>{{ language }}</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="mt-4">
+                    <h3>
+                        <b>{% translate "Pages to be translated" %}(<span id="machine_translation_overlay_pages_number"></span>/<span id="machine_translation_overlay_pages_total_number"></span>)</b>
+                    </h3>
+                    <div class="mt-2">
+                        <div id="machine-translation-overlay-pages-no-pages-warning"
+                             class="hidden bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 mb-5">
+                            {% translate "None of the selected pages fullfills the required HIX score of " %}  {{ hix_threshold }}.
```

Comment:
Also here the variable should be included in the translation text since it's part of the same sentence.